ctagsdotd/haskell.ctags

```diff
@@ -1,8 +0,0 @@
-# See additional-language.ctags for maintaining this file
```

Comment:
TODO: See if the built-in parser is less powerful than this. If so we can enhance it with what it's missing.